security/server/ipsec_vici.go

```diff
@@ -105,6 +105,74 @@ type rekey_connection struct {
 	Reauth     bool   `vici:"reauth"`
 }
 
+type list_sas struct {
+	Ike        string `vici:"ike"`
+	IkeId      string `vici:"ike"`
```

Comment:
Probably doesn't matter, but should this be `vici:"ike-id"` instead?

Reply:
Good catch Steve!